Paymetheus/Overview.xaml

```diff
@@ -1,143 +1,108 @@
-                <RowDefinition Height="Auto"/>
-            </Grid.RowDefinitions>
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition/>
-            </Grid.ColumnDefinitions>
-            <TextBlock Grid.Row="0" HorizontalAlignment="Left" Margin="0 0 0 20" Text="Totals" VerticalAlignment="Top" Foreground="#FF0C1E3E" FontSize="20"/>
-            <Border Grid.Row="1" Margin="0 0 40 0" BorderBrush="#FFA9B4BF" BorderThickness="0 0 0 1" VerticalAlignment="Stretch">
-                <Grid VerticalAlignment="Top">
-                    <Grid.Resources>
-                        <Style TargetType="{x:Type local:AmountLabel}">
-                            <Setter Property="FontSizeWhole" Value="28"/>
-                            <Setter Property="FontSizeDecimal" Value="11"/>
-                            <Setter Property="ForegroundWhole" Value="#FF0C1E3E"/>
-                            <Setter Property="ForegroundDecimal" Value="#FF0C1E3E"/>
-                            <Setter Property="FontWeightWhole" Value="Bold"/>
-                        </Style>
-                    </Grid.Resources>
-                    <Grid.ColumnDefinitions>
-                        <ColumnDefinition/>
-                        <ColumnDefinition/>
-                        <ColumnDefinition/>
-                    </Grid.ColumnDefinitions>
-                    <Border Grid.Column="0" BorderBrush="#FFA9B4BF" BorderThickness="0 0 1 0" Padding="0 0 19 0">
-                        <Grid>
-                            <Grid.RowDefinitions>
-                                <RowDefinition Height="Auto"/>
-                                <RowDefinition Height="Auto"/>
-                            </Grid.RowDefinitions>
-                            <local:AmountLabel Margin="0" Grid.Row="0" HorizontalAlignment="Right" Value="{Binding Path=SynchronizerViewModel.TotalBalance, Source={StaticResource ViewModelLocator}}"/>
-                            <TextBlock Grid.Row="1" HorizontalAlignment="Right" Foreground="#FF0C1E3E" FontSize="11">
-                                grand total in
-                                <Run FontWeight="Bold" Text="{Binding AccountsCount, FallbackValue=0}"/>
-                                <Run FontWeight="Bold" Text="accounts"/>
-                            </TextBlock>
-                        </Grid>
-                    </Border>
 
-                    <Grid Grid.Column="2">
+    <Grid Margin="40 60 0 0">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="60"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="1*"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition/>
+        </Grid.ColumnDefinitions>
+        <TextBlock Grid.Row="0" HorizontalAlignment="Left" Margin="0 0 0 20" Text="Totals" VerticalAlignment="Top" Foreground="#FF0C1E3E" FontSize="20"/>
```

Comment:
while i agree this wasn't actually change as part of the pr.  it just looks that way because lol xml diffs.